easieR.code

```diff
@@ -7,35 +6,52 @@
 
 
 ### easieR
-# Ã  faire : inclure dans easier l'installation  et la mise Ã  jour des packages
+# À faire : inclure dans easier l'installation et la mise à jour des packages
```

Comment:
il faut revoir les accents, ça marche pas sur windows ... (ça marche sur l'un ou sur l'autre, mais manifestemment, il ne veut pas sur les deux. 
Vérifie pack.to.inst qui devient pack.uninst